app/src/main/res/menu/utility_menu.xml

```diff
@@ -6,21 +6,28 @@
         android:id="@+id/import_data"
         android:title="@string/import_dataset"
         android:orderInCategory="100"
-        app:showAsAction="ifRoom"
+        app:showAsAction="always"
         android:icon="@drawable/ic_import" />
 
+    <item
+        android:id="@+id/play_graph"
+        android:title="@string/play_graph"
+        android:orderInCategory="200"
+        app:showAsAction="always"
+        android:icon="@drawable/ic_play" />
+
     <item
         android:id="@+id/stop_data"
         android:title="@string/stop_graph"
-        android:orderInCategory="200"
-        app:showAsAction="ifRoom"
+        android:orderInCategory="300"
+        app:showAsAction="always"
```

Comment:
We'll make `Stop` visible only if it's playing. Otherwise too many showAsAction icons.

Reply:
true :+1: